### ISSUE TITLE
[Feature]Subscribe FreshRSS's logs itself as rss

### ISSUE THREAD

**Issue by WestXu** [marienfressinaud/FreshRSS#6839]
Sometimes sources of some subscription silently failed, only make me realized weeks after, when it occurs to me that I haven't read news from some sources for a long time. Then I check the logs and find out the source deprecated. 

It would be nice if we can subscribe the logs itself, specifically an endpoint to turn the logs from this page `/i/?a=logs` to an rss format.

**Comment by Alkarex** [comment]
Good idea and actually, I think an RSS output of `/?c=subscription` together with the settings and state of the feeds could be easier

**Comment by WestXu** [comment]
That sounds like a lot of work. Logs are simpler to implement I think and easier to filter by the user, and logs contains more info like diagnostics of the system, like once my disk was full and I only realized it after looking at the logs.

**Comment by shedtepherd** [comment]
I would also find this feature useful. I am available for testing if you need any.
